domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/iay/tjeneste/OppgittOpptjeningRestTjeneste.java

```diff
@@ -96,6 +96,40 @@ public Response lagreOppgittOpptjening(@NotNull @TilpassetAbacAttributt(supplier
         return response;
     }
 
+    @POST
+    @Path("/overstyr")
+    @Operation(description = "Lagrer ned mottatt oppgitt opptjening", tags = "oppgitt opptjening", responses = {
+            @ApiResponse(description = "Oppdatert grunnlagreferanse",
+                    content = @Content(mediaType = "application/json",
+                            schema = @Schema(implementation = UuidDto.class)))
+    })
+    @BeskyttetRessurs(action = CREATE, resource = SØKNAD)
```

Comment:
Hvem skal overstyre er det en saksbehandler? Eller kommer dette fra fordel? Hvis dette er en endring på eksisterende opptjening bør man bruke UPDATE og utvide eksisterende ABAC regler med en update operasjon for InterneBrukere.

Reply:
Lag en sak på dette så fikser vi i etterkant ellers vil det feile med update da det ikke finnes en regel i abac ennå.

Reply:
denne er nok feil ja, copy paste, skal fikse, er saksbehnalder som skal endre ja